client/blocks/daily-post-button/test/index.jsx

```diff
@@ -1,43 +1,40 @@
 /** @format */
+jest.mock( 'reader/stats', () => ( {
+	pageViewForPost: () => {},
+	recordAction: () => {},
+	recordGaEvent: () => {},
+	recordTrackForPost: () => {},
+} ) );
+jest.mock( 'lib/analytics', () => ( {
```

Comment:
We should rather use Redux actions to trigger analytics. They won't try to trigger network requests when executing tests. We can also mock out lib/analytics and override some methods on the global level.